wicker/core/storage.py

```diff
@@ -463,17 +487,19 @@ class S3PathFactory(WickerPathFactory):
                     - part-1-attempt-2345.parquet
     """
 
-    def __init__(self, s3_root_path: Optional[str] = None) -> None:
+    def __init__(self, s3_root_path: Optional[str] = None, prefix_replace_path: str = "") -> None:
```

Comment:
@convexquad I know you love sorting but I don't know if users are passing args instead of kwargs so I can't sort these safely.

Reply:
No worries